double/double.mbti

```diff
@@ -37,6 +37,7 @@ impl Double {
   pow(Double, Double) -> Double
   round(Double) -> Double
   signum(Double) -> Double
+  to_bytes(Double) -> Bytes
```

Comment:
can you also move changes about `T::to_bytes` to another PR, along with `bytes_to_t`?